server/schedulers/balance_leader.go

```diff
@@ -357,54 +369,66 @@ func (l *balanceLeaderScheduler) Schedule(cluster schedule.Cluster) []*operator.
 	scoreFunc := func(store *core.StoreInfo) float64 {
 		return store.LeaderScore(plan.kind.Policy, plan.GetOpInfluence(store.GetID()))
 	}
-	sourceCandidate := newCandidateStores(filter.SelectSourceStores(stores, l.filters, cluster.GetOpts()), false, scoreFunc)
-	targetCandidate := newCandidateStores(filter.SelectTargetStores(stores, l.filters, cluster.GetOpts()), true, scoreFunc)
+	storesCandidate := newCandidateStores(filter.SelectSourceStores(stores, l.filters, cluster.GetOpts()), scoreFunc)
```

Comment:
There is a little difference between select source and target when filtering.

Reply:
ono

Reply:
> There is a little difference between select source and target when filtering.

This PR should be closed for the previous reason.